csfieldguide/chapters/content/en/glossary/substitution-cipher.md

```diff
@@ -1,6 +1,6 @@
 # Substitution cipher
 
 A type of cipher that works simply by replacing each letter or combination of letters in a plain text with a certain other letter or combination of letters to make up the cipher text.
```

Comment:
I think this is a bit too long for a glossary entry. The first sentence by itself seems enough.